components/dropdown/dropdown-popover-mixin.js

```diff
@@ -272,6 +272,22 @@ export const DropdownPopoverMixin = superclass => class extends LocalizeCoreElem
 		super.open(opener, applyFocus);
 	}
 
+	// todo: remove this method when removing GAUD-7472-dropdown-popover flag (d2l-filter calls this)
+	requestRepositionNextResize() {
```

Comment:
This is called by `d2l-filter` but it does not appear to be needed with the new implementation.